### PR TITLE
Feature/notification

### PR DESCRIPTION
**Pull request by allenazamoras**

### CODE REVIEW

File: livestream/permissions.py [hunk 1]
```diff
@@ -38,17 +38,17 @@ class AppealsViewSetPermissions(BasePermission):
     Appeal Helper       ['retrieve']
     '''
     def has_object_permission(self, request, view, obj):
-
-        if view.action in ['create', 'list']:
+        print(request.user, obj.owner)
```
Comment (aldnav) [inline livestream/permissions.py]: @allenazamoras you missed this :)

File: userprofile/views.py [hunk 2]
```diff
@@ -82,8 +98,121 @@ def create(self, request):
 
 
 class SearchListView(generics.ListAPIView):
-    queryset = Appeal.objects.filter(status='AVAILABLE')
-    serializer_class = AppealSerializer
+    queryset = Appeal.objects.filter(status='a')
+    print(queryset)
```
Comment (aldnav) [inline userprofile/views.py]: also this :)

File: userprofile/views.py [hunk 3]
```diff
@@ -82,8 +98,121 @@ def create(self, request):
...
+            notif = Notification(message=message1, user=obj.helper,
+                                 icon=icon[status])
+            notif_list.append(notif)
+            message2 = "You " + status + " " + obj.helper.username +\
+                       "'s offer to help."
+            notif = Notification(message=message2, user=obj.appeal.owner,
+                                 icon=icon[status])
+            notif_list.append(notif)
+
+        Notification.objects.bulk_create(notif_list)
```
Comment (aldnav) [inline userprofile/views.py]: @allenazamoras :+1:

File: userprofile/views.py [hunk 4]
```diff
@@ -82,8 +98,120 @@ def create(self, request):
 
 
 class SearchListView(generics.ListAPIView):
-    queryset = Appeal.objects.filter(status='AVAILABLE')
-    serializer_class = AppealSerializer
+    queryset = Appeal.objects.filter(status='a')
```
Comment (jonathantumulak) [inline userprofile/views.py]: @allenazamoras also use the status constants here

**Comment by aldnav** [comment]
:100:

**Pull request merged**